### PR TITLE
Adding bhvalent keymap to crkbd

### PR DESCRIPTION
**Pull request by bhvalent**
## Description

Adding a keymap for crkbd

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/bhvalent/config.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+//#define QUICK_TAP_TERM 0
+//#define TAPPING_TERM 100
+
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/bhvalent/config.h:33]:
```suggestion
```

File: keyboards/crkbd/keymaps/bhvalent/config.h [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    #define RGBLIGHT_EFFECT_KNIGHT
+    #define RGBLIGHT_EFFECT_CHRISTMAS
+    #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+    #define RGBLIGHT_EFFECT_RGB_TEST
+    #define RGBLIGHT_EFFECT_ALTERNATING
+    #define RGBLIGHT_EFFECT_TWINKLE
+    #define RGBLIGHT_LIMIT_VAL 120
+    #define RGBLIGHT_HUE_STEP 10
+    #define RGBLIGHT_SAT_STEP 17
+    #define RGBLIGHT_VAL_STEP 17
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/bhvalent/config.h:48]:
```suggestion
```

File: keyboards/crkbd/keymaps/bhvalent/config.h [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#endif
+
+#define OLED_FONT_H "keyboards/crkbd/lib/glcdfont.c"
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/bhvalent/config.h:51]:
```suggestion
```

File: keyboards/crkbd/keymaps/bhvalent/keymap.c [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include <stdio.h>
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/bhvalent/keymap.c:20]:
```suggestion
```

File: keyboards/crkbd/keymaps/bhvalent/keymap.c [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+    return false;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  if (record->event.pressed) {
+    set_keylog(keycode, record);
+  }
+  return true;
+}
+#endif // OLED_ENABLE
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/bhvalent/keymap.c:176]:
```suggestion
```

File: keyboards/crkbd/keymaps/bhvalent/rules.mk
Comment (waffle87) [inline keyboards/crkbd/keymaps/bhvalent/rules.mk:1]: This file should be removed, as these features are all already enabled by default for this keyboard.

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
Unfortunately since this PRs creation, QMK has [changed its stance](https://docs.qmk.fm/#/ChangeLog/20230827?id=notable-changes) on accepting user keymaps to the repository.

> qmk/qmk_firmware is no longer accepting PRs for keymaps other than for manufacturer-supported keymaps. User keymap workflow has been documented [here](https://docs.qmk.fm/#/newbs) for several years. This change is to progressively reduce the maintenance burden on the project, and to allow us to focus on the core features of QMK.

In the near future we will be providing a more user focused workflow for keymaps, but till then the suggestion is to keep changes in your own fork.

While we appreciate the time that was taken on this PR, hopefully you understand we are closing it, so that QMK can improve in the long term.

**Pull request closed without merging**